tests/impact/client/fixtures.py

```diff
@@ -233,6 +233,87 @@ def import_fmu(sem_ver_check, mock_server_base):
     )
 
 
+@pytest.fixture
+def upload_result(sem_ver_check, mock_server_base):
+    json = {
+        "data": {
+            "id": "2f036b9fab6f45c788cc466da327cc78workspace",
+            "status": "ready",
+            "error": "None",
+        }
+    }
+
+    return with_json_route(mock_server_base, 'POST', 'api/uploads/results', json)
+
+
+@pytest.fixture
+def upload_result_status_ready(sem_ver_check, mock_server_base):
+    json = {
+        "data": {
+            "id": "2f036b9fab6f45c788cc466da327cc78workspace",
+            "status": "ready",
+            "data": {
+                "resourceUri": "api/external-result/2f036b9fab6f45c788cc466da327cc78workspace"
+            },
+            "error": {"message": "Something.", "code": 12015},
+        }
+    }
+    return with_json_route(
+        mock_server_base,
+        'GET',
+        'api/uploads/results/2f036b9fab6f45c788cc466da327cc78workspace',
+        json,
+    )
+
+
+@pytest.fixture
+def upload_result_status_running(sem_ver_check, mock_server_base):
+    json = {
+        "data": {
+            "id": "2f036b9fab6f45c788cc466da327cc78workspace",
+            "status": "running",
+            "data": {
+                "resourceUri": "api/external-result/2f036b9fab6f45c788cc466da327cc78workspace"
+            },
+            "error": {"message": "Something.", "code": 12015},
+        }
+    }
+
+    return with_json_route(
+        mock_server_base,
+        'GET',
+        'api/uploads/results/2f036b9fab6f45c788cc466da327cc78workspace',
+        json,
+    )
+
+
+@pytest.fixture
+def upload_result_meta(sem_ver_check, mock_server_base):
+    json = {
+        "id": "2f036b9fab6f45c788cc466da327cc78workspace",
+        "createdAt": 1588016253,
+        "name": "result_for_PID",
+        "description": "This is a result file for PID controller",
+        "workspaceId": "workspace",
+    }

```

Comment:
I think the API would return {'data': ...}, this was my point with one of the issues during release testing. Can you dubble check it?